<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminHomeWindowN</class>
 <widget class="QMainWindow" name="AdminHomeWindowN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>782</width>
      <height>457</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Добавить пользователя</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Фамилия</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Имя</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Отчество</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Адрес</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Телефон</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Должность</string>
      </property>
      <item>
       <property name="text">
        <string>Покупатель</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Поставщик</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Администратор</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="regUser">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>340</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>Зарегистрировать</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Пароль</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Удалить пользователя</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>История покупок</string>
     </attribute>
     <widget class="QTableView" name="purchaseHistoryTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>История изменения цен</string>
     </attribute>
     <widget class="QTableView" name="priceChangeHistoryTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>782</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
